language/move-prover/spec-lang/src/translate.rs

```diff
@@ -1168,6 +1173,17 @@ impl<'env, 'translator> ModuleTranslator<'env, 'translator> {
 
         // Perform post reduction of module invariants.
         self.reduce_module_invariants();
+
+        // Warn about unused schemas.
+        for (schema_name, loc) in &self.parent.unused_schema_table {
+            self.parent.env.warn(
+                loc,
+                &format!(
+                    "A schema is not used: {}",
```

Comment:
I suggest to make the message "unused schema `{}`". We usually start error messages with lowercase and use backquotes for references to symbols.

Reply:
I was actually following the format of the error message below "A postcondition might not hold on this return path." Is this error message directly from Boogie? Maybe we should standardize the format of error messages too.